libcxx/utils/ci/docker-compose.yml

```diff
@@ -1,11 +1,38 @@
 version: '3.7'
+
+x-versions: &compiler_versions
+  GCC_LATEST_VERSION: 13
+  LLVM_HEAD_VERSION: 18
+
 services:
-  libcxx-builder:
-    image: ghcr.io/libcxx/libcxx-builder
+  buildkite-builder:
+    image: ghcr.io/libcxx/buildkite-builder:testing
     build:
       context: .
       dockerfile: Dockerfile
+      target: buildkite-builder
       args:
-        GCC_LATEST_VERSION: 13
-        # LLVM POST-BRANCH bump version
-        LLVM_HEAD_VERSION: 18
+        BASE_IMAGE: ubuntu:jammy
+        <<: *compiler_versions
+  actions-builder:
+    image: ghcr.io/libcxx/actions-builder:testing
+    build:
+      context: .
+      dockerfile: Dockerfile
+      target: actions-builder
+      args:
+        BASE_IMAGE: ghcr.io/actions/actions-runner:latest
+        <<: *compiler_versions
+  android-buildkite-builder:
+    image: ghcr.io/libcxx/android-buildkite-builder:testing
+    build:
+      context: .
+      dockerfile: Dockerfile
+      target: android-buildkite-builder
+      args:
+        BASE_IMAGE: ubuntu:jammy
+        ANDROID_CLANG_VERSION: r498229b
+        ANDROID_CLANG_PREBUILTS_COMMIT: 5186d132c99aa75dc25207c392e3ea5b93d0107e
+        ANDROID_SYSROOT_BID: 10957860
+        <<: *compiler_versions
+
```

Comment:
Not attached to this line: I think we need to update `libcxx/utils/ci/run-buildbot-container` as well.